src/utils/num.ts

```diff
@@ -60,23 +106,48 @@ export function toStorageKey(number: BigNumberish): string {
  * Convert hexadecimal string to decimal string
  * @param hex hex-string
  * @returns format: decimal string
+ * @example
+ * ```typescript
+ * const hexString = "0x2fd23d9182193775423497fc0c472e156c57c69e4089a1967fb288a2d84e914";
+ * const result = hexToDecimalString(hexString);
+ * // result = "32507161997631881240494522159444018041090212371621416251492750949915267618964";
+ * ```
  */
 export function hexToDecimalString(hex: string): string {
   return BigInt(addHexPrefix(hex)).toString(10);
 }
 
 /**
- * Remove hex string leading zero and lowercase it
- * @example '0x01A...' -> '0x1a..'
+ * Remove hex string leading zero and lowercase it;
  * @param hex hex-string
  * @returns format: hex-string
+ * @example
+ * ```typescript
+ * const hexString = '0x01A...'
+ * const result = cleanHex(hexString);
+ * //result = '0x1a..'
```

Comment:
You should use a real hex, not with the ...

Reply:
changed '0x01A...' to '0x01A2F3' and also '0x1a...' to '0x1a2f3.

add space after each '//' in entire doc